docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: "3"
+
+services:
+  verifier:
+    image: ethereumoptimism/go-ethereum:latest
+    volumes:
+      - verifier:/root/.ethereum:rw
+    env_file:
+      - docker-compose.env
+    environment:
+      - ROLLUP_VERIFIER_ENABLE=true
+      - CHAIN_ID=420
```

Comment:
If we want to use this for our fraud proof security drill then we should change this to 69